app/views/events/participants.html.haml

```diff
@@ -0,0 +1,24 @@
+.col-md-12
+  %h3 Participants of #{@event.name}
+  %br
+  %table.table
+    %tr
+      %th #
+      %th Nickname
+      %th Email
+      %th State
+      %th Role
+      %th Change role to event organizer
+    - @event.requests.map(&:user).uniq.each_with_index do |user,index|
```

Comment:
You could use ``with_index(1)`` so you don't need to add +1 in the loop.
https://apidock.com/ruby/Enumerator/with_index

Also I don't really like the query and mapping etc. I would prefer to do that in the view in a proper way.

```
User.where(request: @event).includes(:request).distinct.
```

This would also trigger several n+1 queries
http://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations